### PR TITLE
Fix/footer links

### PR DESCRIPTION
**Pull request by rafinskipg**
Change some links on the footer to add our discord link, removed foundation

image

### CODE REVIEW

**Comment by b-pmcg** [comment]
lgtm, what about adding the discord icon to the bottom row of icons as well?

**Review by b-pmcg — APPROVED** [review]

**Comment by adamgoth** [comment]
> lgtm, what about adding the discord icon to the bottom row of icons as well?

Agree that the icon would be nice, goes with the other social

File: components/Footer/LongFooter.tsx [hunk 1]
```diff
@@ -78,11 +78,19 @@ export default function LongFooter({ locale = 'en' }: { locale?: string }): Reac
...
           url: 'https://makerdao.com/contact',
-          title: t('Contact')
+          title: t('Press & media')
+        },
+        {
+          url: 'https://discord.gg/2sWcgCDWCX',
+          title: 'Governance UI Discord'
+        },
+        {
+          url: 'https://chat.makerdao.com/',
```
Comment (adamgoth) [inline components/Footer/LongFooter.tsx:92]: There's a rocket chat link in social too

Comment (rafinskipg) [reply components/Footer/LongFooter.tsx:92]: yeep , but it was more in case uses "CTRL + F" search for contact or whatever

**Pull request merged**